pprof.go

```diff
@@ -24,6 +24,8 @@ import (
 
 	"github.com/chzyer/readline"
 	"github.com/google/pprof/driver"
+
+	_ "net/http/pprof" // Enable profiling pprof itself in the web UI mode.
```

Comment:
In the past I have wanted to profile startup performance itself. This approach does not allow profiling startup time. An alternative would be flags (separate ones will be needed for cpu and memory profiling).

Not sure how we exit cleanly though to allow the profile to be saved if we are using flags.